scripts/setup/upgrade-postgresql

```diff
@@ -26,6 +26,24 @@ if [[ "$UPGRADE_TO" -lt "$UPGRADE_FROM" ]]; then
     exit 1
 fi
 
+# Verify that the version in /etc/zulip/zulip.conf is the version that
+# Django actually stores its data in
+DATA_IS_IN=$(su zulip -c '/home/zulip/deployments/current/manage.py shell -c "from django.db import connection; print(int(connection.cursor().connection.server_version/10000))"')
```

Comment:
Hmm, will this not work unless the current system is both the application server and the postgres server? 

Would it make sense to instead just check certain tables exist in the postgres instance in the case that there's no `deployments/current` on the current system?

Reply:
I guess my proposal would be to:
* Only do this check in the event that there's a `/home/zulip/deployments/current` on the system, or ideally some better sign involving `/etc/zulip/zulip.conf` that it's also a Zulip application server.
* Maybe do a `psql` based query for whether a `zerver_realm` table exists in the database we think has a zulip server in it  
 to upgrade before attempting to do something with that database?

Reply:
I considered the psql solution, but it's hard to know how to connect to the PostgreSQL instance automatically.  We do it in `process_fts_update` and it's pretty fiddly.

So I don't know the additional work is worth it here?

Reply:
Yeah I think it's probably fine to have this just cover the "you also have a zulip server on this system" case given the other one is annoying.